src/renderer/js/component/uriIndicator/view.jsx

```diff
@@ -60,7 +60,13 @@ class UriIndicator extends React.PureComponent {
 
     const inner = (
       <span>
-        <span className={`channel-name ${smallCard && "channel-name--small"}`}>
+        <span
+          className={
+            "channel-name" +
```

Comment:
can you use `classnames` for this? https://github.com/lbryio/lbry-app/blob/610814160572a51840366eff7e604e418c58b353/src/renderer/js/component/address/view.jsx#L24

Reply:
Yes, this seems awesome.